schema-lending.graphql

```diff
@@ -994,6 +994,9 @@ type Liquidate implements Event @entity {
   " Address that carried out the liquidation "
   from: String!
 
+  " Address that got liquidated "
+  liquidated: String!
```

Comment:
does `liquidatee` sound less ambiguous? `liquidated` could also mean the liquidated asset aka collateral

Reply:
I don't think `liquidatee` is a real word :)

Reply:
Also, `liquidated` is a bit ambiguous

Reply:
or just `liquidatedAccount` or even `victim` 😆